<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsNewVectorLayerDialogBase</class>
 <widget class="QDialog" name="QgsNewVectorLayerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Vector Layer</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="mFileEncoding">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mFileFormatLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>File format</string>
     </property>
     <property name="buddy">
      <cstring>mFileFormatComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="mFileFormatComboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mFileFormatLabel_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>File encoding</string>
     </property>
     <property name="buddy">
      <cstring>mFileFormatComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QgsProjectionSelectionWidget" name="mCrsSelector" native="true">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="mPointRadioButton">
        <property name="text">
         <string>Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mLineRadioButton">
        <property name="text">
         <string>Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mPolygonRadioButton">
        <property name="text">
         <string>Polygon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>New attribute</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>mNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="mNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>mTypeBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QComboBox" name="mTypeBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width</string>
        </property>
        <property name="buddy">
         <cstring>mWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="mWidth"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Precision</string>
        </property>
        <property name="buddy">
         <cstring>mPrecision</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="mPrecision"/>
      </item>
      <item row="4" column="4">
       <widget class="QToolButton" name="mAddAttributeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Add attribute to list</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Add to attributes list</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../images/themes/default/mActionNewAttribute.png</normaloff>../../images/themes/default/mActionNewAttribute.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Attributes list</string>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QTreeWidget" name="mAttributeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Width</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Precision</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>121</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="mRemoveAttributeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Delete selected attribute</string>
        </property>
        <property name="text">
         <string>Remove attribute</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../images/themes/default/mActionDeleteAttribute.png</normaloff>../../images/themes/default/mActionDeleteAttribute.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">qgsprojectionselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mPointRadioButton</tabstop>
  <tabstop>mLineRadioButton</tabstop>
  <tabstop>mPolygonRadioButton</tabstop>
  <tabstop>mFileEncoding</tabstop>
  <tabstop>mFileFormatComboBox</tabstop>
  <tabstop>mCrsSelector</tabstop>
  <tabstop>mNameEdit</tabstop>
  <tabstop>mTypeBox</tabstop>
  <tabstop>mWidth</tabstop>
  <tabstop>mPrecision</tabstop>
  <tabstop>mAddAttributeButton</tabstop>
  <tabstop>mAttributeView</tabstop>
  <tabstop>mRemoveAttributeButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsNewVectorLayerDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsNewVectorLayerDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
